calendar.js

```diff
@@ -9,16 +9,32 @@ function myTable() {
         table.appendChild(th);
     }
     //function to display hourly blocks of each day for one week
+    var time = [];
+    $.getJSON('calendar.json', function (data) {
+        $.each(data, function (i, item) {
+            time[i] = item.time.length
+            })
+    })
     for (var i = 0; i < 24; i++) {
+        
```

Comment:
extract the logic of updating the calendar schedule to it's own function so we can test it separately from the initial calendar display code